src/Cryptol/Backend.hs

```diff
@@ -813,3 +815,7 @@ type FPArith2 sym =
   SFloat sym ->
   SFloat sym ->
   SEval sym (SFloat sym)
+
+wordLen :: forall sym. Backend sym => sym -> SWord sym -> Integer
+wordLen _ x = wordLen' (Nothing :: Maybe sym) x
```

Comment:
Similarly, I wonder if we should pass `Proxy @sym` instead of `Nothing :: Maybe sym` here (and elsewhere in this PR where we use `Nothing :: Maybe sym`.